src/test/java/foodwhere/logic/commands/CommandTestUtil.java

```diff
@@ -28,21 +28,33 @@ public class CommandTestUtil {
     public static final String VALID_NAME_BOB = "Bob Choo";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_DATE_AMY = "1/1/2020";
-    public static final String VALID_DATE_BOB = "31/12/2021";
+    public static final String VALID_DATE_AMY = "2022-09-20";
```

Comment:
Perhaps we can change the date format to either `DD/MM/YYYY` or `DD-MM-YYYY` as discussed?